tests/Doctrine/Tests/DBAL/Functional/ExceptionTest.php

```diff
@@ -246,9 +246,10 @@ public function testConnectionExceptionSqLite($mode, $exceptionClass)
             $this->markTestSkipped("Only fails this way on sqlite");
         }
 
-        $filename = sprintf('%s/%s', sys_get_temp_dir(), 'doctrine_failed_connection.db');
+        $filename = sprintf('%s/%s', sys_get_temp_dir(), 'doctrine_failed_connection_'.$mode.'.db');
```

Comment:
@Tobion can we please try not to mix things in this PR? These test fixes are not really related to the code changes and rather try to make out test suite compatible with Windows. Please let's deal with it in another PR to keep things focused and clean.